c2corg_ui/templates/index.html

```diff
@@ -1,4 +1,53 @@
+<%!
+from c2corg_common.attributes import activities
+%>
 <%inherit file="base.html"/>
 
-<h1 translate class="welcome-title text-center">Welcome to Camptocamp.org</h1>
-<app-map></app-map>
+<div class="home-page">
+  <section ng-hide="userCtrl.auth.isAuthenticated()" class="site-intro not-logged">
+    <div class="intro-banner">
+      <h1 class="welcome-title text-center" translate>The mountain sports community</h1>
+
+      <div class="route-activities text-center">
+        % for activity in activities:
+        <a href="${request.route_url('outings_index')}#&act=${activity}">
+          <span class="activity icon-${activity}" uib-tooltip="{{'${activity}' | translate}}"></span>
+        </a>
+        % endfor
+      </div>
+
+      <p class="text-center" translate>Camptocamp.org aims to
+        facilitate information sharing between
+        mountain addicts and contribute to the safety of mountain activities.
+      </p>
+
+      <div class="intro-shortcuts">
+        <a href="${request.route_url('routes_index')}">
+          <button class="btn green-btn">
+            <span class="glyphicon glyphicon-arrow-right"></span>
+            <span translate>Topoguide</span><br>
+            <span translate class="small">Maps, routes, summits...</span>
```

Comment:
Rather write ``Conditions, summits, routes, huts, climbing sites, images...`` if it's not too long.